azurerm/internal/services/netapp/tests/resource_arm_netapp_volume_test.go

```diff
@@ -437,6 +447,10 @@ resource "azurerm_netapp_volume" "test" {
     unix_read_only    = false
     unix_read_write   = true
   }
+
+  tags = {
+    "foo" = "bar"
```

Comment:
Could we ensure capitals work
```suggestion
    "FoO" = "BaR"
```